src/Sylius/Behat/Resources/config/suites/api/product/managing_products.yml

```
@@ -41,6 +41,7 @@ default:
                 - sylius.behat.context.setup.shipping_category
                 - sylius.behat.context.setup.taxonomy
                 - sylius.behat.context.setup.zone
+                - Sylius\Behat\Context\Api\Admin\ManagingProductTaxonsContext
 
                 - sylius.behat.context.api.admin.managing_products
```

```suggestion

                - sylius.behat.context.api.admin.managing_products
                - Sylius\Behat\Context\Api\Admin\ManagingProductTaxonsContext
```
And similar change should be applied in other suites

---

I Think that Sylius\Behat\Context\Api\Admin\ManagingProductTaxonsContext is not needed in other Suites because it only allows 'When I change that the "T-Shirt" product belongs to the "T-Shirts" taxon' to be used